production/helm/loki/templates/write/statefulset-write.yaml

```
@@ -151,7 +151,7 @@ spec:
       {{- end }}
       {{- with .Values.write.dnsConfig }}
       dnsConfig:
-        {{- tpl . $ | nindent 8 }}
+        {{- toYaml . | nindent 8 }}
```

This should prevent a breaking change:

```suggestion
        {{- if kindIs "map" . }}
        {{- tpl (toYaml .) $ | nindent 8 }}
        {{- else }}
        {{- tpl . $ | nindent 8 }}
        {{- end }}
```